### DISCUSSION TITLE
NavLinks do not work unless SSR, Anyone Else?

### DISCUSSION THREAD

**Discussion by Aquaritek** [dotnet/aspnetcore#51661] [Q&A]
Before I open a ticket. Does anyone else experience NavLink components becoming activity inert once they're rendered in any Interactivity mode?

The links are clickable but any matching or automated active state engagement is no longer operational after made Interactive.

Anyone else?

**Comment by NitroDH** [comment]
Yeah we have this issue. NavLink and even in component libraries like MudNavLink do not work at all unless rendered in SSR. Furthermore the NavigationManager specfically OnLocationChanged events do nothing in Interactive Components.
